src/scss/layout/_title.scss

```diff
@@ -52,3 +52,16 @@
     }
   }
 }
+
+span.first {
+  display: block;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: var(--h3);
+  font-weight: 300;
+}
+
+span.second {
+  @include config.font-family('amp');
+
+  font-style: italic; // needed for Safari
+}
```

Comment:
Is this needed at all? Can we apply just the italics to the one word we want italicized?

Reply:
the `<em>` tag applies different styling to what i would expect; it currently makes the text look superscripted?

Reply:
oh interesting, I wonder where that's coming from. Have you looked at the CSS to see what's causing that?

Generally, we should avoid writing extra markup to work around unintentional styles - and instead try to understand and address where that style is coming from.